students/Deniss_Semcovs/Lesson02/printing_a_grid.py

```diff
@@ -0,0 +1,32 @@
+
+print("+"+" -"*4+" +"+" -"*4+" +")
+print("|"+"  "*4+" |"+"  "*4+" |")
+print("|"+"  "*4+" |"+"  "*4+" |")
+print("|"+"  "*4+" |"+"  "*4+" |")
+print("|"+"  "*4+" |"+"  "*4+" |")
+print("+"+" -"*4+" +"+" -"*4+" +")
+print("|"+"  "*4+" |"+"  "*4+" |")
+print("|"+"  "*4+" |"+"  "*4+" |")
+print("|"+"  "*4+" |"+"  "*4+" |")
+print("|"+"  "*4+" |"+"  "*4+" |")
+print("+"+" -"*4+" +"+" -"*4+" +")
+
+# or
+
+print("""
++ - - - - + - - - - +
+|         |         |
+|         |         |
+|         |         |
+|         |         |
++ - - - - + - - - - +
+|         |         |
+|         |         |
+|         |         |
+|         |         |
++ - - - - + - - - - +
+""") 
+
+# or
+
+print("+"+" -"*4+" +"+" -"*4+" +"),("|"+"  "*4+" |"+"  "*4+" |"),("|"+"  "*4+" |"+"  "*4+" |")
```

Comment:
Hmm...don't think this is working as expected.  You may want to take another look.